### PR TITLE
px4-dev-ros-kinetic: upgrade to python3 to meet matplotlib requirements

### PR DESCRIPTION
**Pull request by TSC21**
In order to fix http://ci.px4.io:8080/blue/organizations/jenkins/PX4%2Fcontainers/detail/master/22/pipeline/144

### CODE REVIEW

File: docker/px4-dev/Dockerfile_ros-kinetic [hunk 1]
```diff
@@ -20,6 +20,8 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key 421C365BD9F
 		protobuf-compiler \
 		python-catkin-tools \
 		python-tk \
```
Comment (dagar) [inline docker/px4-dev/Dockerfile_ros-kinetic]: Can these change as well?

**Comment by dagar** [comment]
Can we try to get each container to only contain a single python version? It removes a lot of the confusion.

File: docker/px4-dev/Dockerfile_ros-kinetic (hunk 1 above)
Comment (TSC21) [reply docker/px4-dev/Dockerfile_ros-kinetic]: `python-catkin-tools` you can't.

**Comment by TSC21** [comment]
> Can we try to get each container to only contain a single python version? It removes a lot of the confusion.

You can't because there are tools that are Python2 dependent and others Python3 dependent. The official supported Python version for ROS Kinetic for example is Python2.7

**Comment by TSC21** [comment]
Unless we downgrade `matplotlib` from 3.0, we do require two Python versions on the containers.

File: docker/px4-dev/Dockerfile_ros-kinetic (hunk 1 above)
Comment (TSC21) [reply docker/px4-dev/Dockerfile_ros-kinetic]: Actually, I don't even know why we need Tkinter here. Do you know why?

Comment (dagar) [reply docker/px4-dev/Dockerfile_ros-kinetic]: The `python3-catkin-tools` package doesn't work?

Comment (dagar) [reply docker/px4-dev/Dockerfile_ros-kinetic]: Tkinter might be needed for the ecl analysis script. https://github.com/PX4/Firmware/tree/master/Tools/ecl_ekf

Comment (TSC21) [reply docker/px4-dev/Dockerfile_ros-kinetic]: > The `python3-catkin-tools` package doesn't work?

Not saying it doesn't work, but this tool is used exclusively for ROS builds, so it makes sense to me that the tools used in ROS remain synced on the same Python version.

**Pull request merged**